### ISSUE TITLE
中文字体显示问题

### ISSUE THREAD

**Issue by OKFred** [neil3d/excel2json#36]
see pic
![image](https://user-images.githubusercontent.com/67962064/98466752-73a3d200-220c-11eb-8cb0-0c40772c0dd9.png)

**Comment by neil3d** [comment]
是的。。。这个是个老问题了。。。不打算修了。因为计划使用 Visual Studio 2022 进行重构，到时候打算使用 MAUI ，也就不用这个控件了。
